Soccar [UI]/src/nl/socnet/message/PlayerSyncMessage.java

```diff
@@ -0,0 +1,65 @@
+package nl.socnet.message;
+
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+import nl.socnet.message.handler.PlayerSyncMessageHandler;
+
+@MessageEvent(id = MessageConstants.SYNC_POSITION_PLAYER_MESSAGE_ID, handler = PlayerSyncMessageHandler.class)
+public final class PlayerSyncMessage extends Message {
+
+    private final int playerId;
+
+    private final float x;
+    private final float y;
+    private final float angle;
+
+    private final float linearVelocityX;
+    private final float linearVelocityY;
+
+    private final float angularVelocity;
+
+    public PlayerSyncMessage(int playerId, float x, float y, float angle, float linearVelocityX, float linearVelocityY, float angularVelocity) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)